dbsync-central/init.prod.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+apk update
+apk upgrade
+apk add bash
+apk add git
+apk add maven
+
+
+#RETRIEVE EXISTING BUILD
+if test -f "$shared_dir/openmrs-eip/app/target/openmrs-eip-app-1.0-SNAPSHOT.jar"; then
+	cp $shared_dir/openmrs-eip/app/target/openmrs-eip-app-1.0-SNAPSHOT.jar $home_dir/openmrs-eip-app.jar
+fi
+
+
+if test ! -f "$home_dir/openmrs-eip-app.jar"; then
+	if test ! -d "$shared_dir/openmrs-eip"; then
+		cd $shared_dir
+		git clone https://github.com/FriendsInGlobalHealth/openmrs-eip.git
+	fi
+
+	cd $shared_dir/openmrs-eip
+	git pull oring master
+
+	mvn clean install -Ppt -DskipTests
```

Comment:
I thought you will be running the builds manually and publishing them to a shared folder.

Reply:
I would also think you should be building a specific revision that way and work in progress in master is not deployed by mistake

Reply:
Okay